src/StreamMaster.hpp

```diff
@@ -9,10 +9,15 @@
 
 #include <algorithm>
 #include <atomic>
+#include <exception>
+#include <queue>
 #include <thread>
 
 #include "DemuxTopic.h"
 #include "FileWriterTask.h"
+#include "KafkaW.h"
+#include "Status.hpp"
+#include "StatusWriter.hpp"
 #include "logger.h"
 #include "utils.h"
 
```

Comment:
I realise that it is not in your changes, but on line 24 there is a declaration (`struct Streamer;`) which should be removed. The `Streamer` being used is in the `FileWriter` namespace so this one must be a mistake.